examples/minipax/minipax.e

```diff
@@ -117,7 +117,7 @@ feature {NONE} -- Implementation
 						across
 							l_dir.entries as l_cur
 						loop
-							if l_cur.item.name /~ "." and l_cur.item.name /~ ".." then
+							if not l_cur.item.is_current_symbol /~ "." and not l_cur.item.is_parent_symbol then
```

Comment:
use the PATH.is_*_symbol .. it also help for portability  (not all platforms use "." and "..")